src/main/java/pl/niewiemmichal/underhiseye/model/LaboratoryExamination.java

```diff
@@ -0,0 +1,65 @@
+package pl.niewiemmichal.underhiseye.model;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@RequiredArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class LaboratoryExamination {
+
+    @Id @GeneratedValue
+    private Integer id;
+
+    @Lob
+    private String note;
+
+    @Lob
+    private String result;
+
+    @NonNull
+    @Column(nullable = false)
+    private Integer status;
```

Comment:
Create [enum](https://www.geeksforgeeks.org/enum-in-java/) with statuses and use it instead of Integer. Enum should have all states, that examination can be in.